packages/block-editor/src/components/rich-text/list-edit.native.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * WordPress dependencies
+ */
+
+import { Toolbar } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import {
+	changeListType,
+	getStartListFormat,
+} from '@wordpress/rich-text';
+
+/**
+ * Internal dependencies
+ */
+
+import BlockFormatControls from '../block-format-controls';
+
+/**
+ * Whether or not the root list is selected.
+ *
+ * @return {boolean} True if the root list or nothing is selected, false if an
+ *                   inner list is selected.
+ */
+function isListRootSelected( value ) {
```

Comment:
Why not replace `getStartListFormat` with both `isListRootSelected` and `isActiveListType`? This seems weird to me.

Reply:
👋 @ellatrix , not sure I follow. Do you mean to completely remove getStartListFormat and implement its pieces inside `isListRootSelected` and `isActiveListType`?

Reply:
Yes, export `isListRootSelected` and `isActiveListType` from `rich-text` as unstable.

Reply:
Done with 4451cb3.